Index: cjkuni-uming-fonts-0.2.20080216.1/64-ttf-arphic-uming.conf
===================================================================
--- cjkuni-uming-fonts-0.2.20080216.1.orig/64-ttf-arphic-uming.conf
+++ cjkuni-uming-fonts-0.2.20080216.1/64-ttf-arphic-uming.conf
@@ -8,9 +8,7 @@
 		<test name="family">
 			<string>serif</string>
 		</test>
-		<edit name="family" mode="prepend" binding="strong">
-			<string>Bitstream Vera Serif</string>
-			<string>DejaVu Serif</string>
+		<edit name="family" mode="prepend">
 			<string>AR PL UMing HK</string>
 			<string>AR PL UMing CN</string>
 		</edit>
@@ -22,9 +20,7 @@
 		<test name="family">
 			<string>sans-serif</string>
 		</test>
-		<edit name="family" mode="prepend" binding="strong">
-			<string>Bitstream Vera Sans</string>
-			<string>DejaVu Sans</string>
+		<edit name="family" mode="prepend">
 			<string>AR PL UMing HK</string>
 			<string>AR PL UMing CN</string>
 		</edit>
@@ -36,9 +32,7 @@
 		<test name="family">
 			<string>monospace</string>
 		</test>
-		<edit name="family" mode="prepend" binding="strong">
-			<string>Bitstream Vera Sans Mono</string>
-			<string>DejaVu Sans Mono</string>
+		<edit name="family" mode="prepend">
 			<string>AR PL UMing HK</string>
 			<string>AR PL UMing CN</string>
 		</edit>
